apps/css-workshop/pages/select.html

```diff
@@ -333,6 +384,91 @@ <h2>Default</h2>
         </div>
       </label>
 
+      <label
+        class="iui-input-grid"
+        data-iui-status="positive"
+      >
+        <div class="iui-input-label">Select label</div>
+        <div class="iui-input-with-icon">
+          <select class="iui-select-button">
```

Comment:
Since you're using "Select native label` in other places, you can use it in the remaining places too.

Reply:
Oops, bad copy / paste, my mistake.  Fixed!